index.js

```diff
@@ -32,6 +34,19 @@ module.exports = {
   paths: paths,
   absolutePaths: absolutePaths,
   treeForVendor: function() {
+
+    var jqueryPath;
+    try {
+      jqueryPath = path.dirname(resolve.sync('jquery/package.json', { basedir: this.project.root }));
+    } catch (error) {
+      jqueryPath = path.dirname(require.resolve('jquery/package.json'));
+    }
```

Comment:
Do we want to do a `finally` here and to make sure we a `jqueryPath` and if not throw?